sdk/resourcemanager/azure-resourcemanager-appservice/src/test/java/com/azure/resourcemanager/appservice/OneDeployTests.java

```diff
@@ -116,6 +136,82 @@ public void canPushDeployJar() throws Exception {
         waitForRuntimeSuccess(slot2, slotDeploymentId);
     }
 
+    // test uses storage account key and connection string to configure the function app
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void canDeployFlexConsumptionFunctionApp() {
```

Comment:
nit, should we put this test into `FunctionAppTests`? As far as I can see, FunctionApp doesn't implement `SupportsOneDeploy`.

Reply:
The endpoint `api/publish` is the OneDeploy endpoint.

I guess my mistake was the naming of `SupportsOneDeploy` interface. From what we see currently, it is only `SupportsWebAppOneDeploy`...

Let me see if I can get some clarification from service, about what is their terminology on `api/publish` endpoint.